packages/sdk/src/helper/UndefinedToNull.ts

```diff
@@ -0,0 +1,7 @@
+export type UnionUndefinedToNull<T> = T extends undefined ? null : T;
+
+export type UndefinedToNull<T> = {
```

Comment:
What does this type do?

Reply:
It will convert all the undefined to null, leave it for future usage